src/test/java/com/google/javascript/clutz/ProgramSubject.java

```diff
@@ -96,7 +76,7 @@ StringSubject diagnosticStream() {
   }
 
   private String[] parse() throws AssertionError {
-    Options opts = new Options(externs == Externs.NONE);
+    Options opts = new Options(withPlatform == false && externFile == null);
```

Comment:
could you add a comment on what the parameter does? `new Options(/* include externs */ ...)?